### ISSUE TITLE
Add its.lemma property to its helper

### ISSUE THREAD

**Issue by pallaviratra** [winkjs/wink-nlp#6]

**Comment by prtksxna** [comment]
Need to make sure we revert commit - https://github.com/winkjs/winkjs.github.io/commit/db3f825a46ce7979dc05ed4545a1aca00aef9eb6 once this is done.

**Comment by LukeGT** [comment]
Would love this feature! Is it coming soon or not a priority right now?

**Comment by rachnachakraborty** [comment]
Hi @LukeGT 

Thank you for your interest in wink-nlp. These features are definitely in our plan to be released soon.

For any immediate need, please use  [wink-lemmatizer](https://www.npmjs.com/package/wink-lemmatizer).

Shall keep you posted.

Best,
Rachna
